<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="nodeProfileWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Node Profile</string>
  </property>
  <property name="windowOpacity">
   <double>0.970000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="addr_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="desc_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Notes:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="desc_lineedit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>100</y>
         <width>320</width>
         <height>20</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
  </widget>
   <widget class="QLabel" name="alias_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Alias:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight</set>
    </property>
   </widget>
      <widget class="QLineEdit" name="alias_lineedit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>140</y>
         <width>320</width>
         <height>20</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
  </widget>
   <widget class="QPushButton" name="saveAliasBtn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>250</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Alias</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="openInWebsiteBtn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>View on Website</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  </widget>
 <resources/>
 <connections/>
</ui>
